app/views/papers/_sort_by.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class="btn-group btn-group-xs nav-button-group sort" role="group" aria-label="...">
+  <%= link_to 'Votes', url_for(sort: 'votes'), class: 'btn btn-default ' + ('active' if params[:sort] != 'pub_date').to_s, role: "button" %>
+  <%= link_to 'Publish Date', url_for(sort: 'pub_date'), class: 'btn btn-default '  + ('active' if params[:sort] == 'pub_date').to_s, role: 'button'  %>
```

Comment:
You can just use string interpolation here:

```ruby
"btn btn-default #{'active' if params[:sort] != 'votes}"
```

Also the `('active' if params[:sort] != 'pub_date').to_s` stuff is kind of silly for multiple reasons. `#to_s` is totally unnecessary, doubly so if you use string interpolation, and then checking if it's "not equal to 'pub_date'" is much less readable and has more side effects than just saying what you want - "make it active if the sort parameter is "votes."